parser/parser_test.go

```diff
@@ -3710,6 +3711,19 @@ func TestDDL(t *testing.T) {
 		{"create resource group x ru_per_sec=3000 burstable", true, "CREATE RESOURCE GROUP `x` RU_PER_SEC = 3000 BURSTABLE"},
 		{"create resource group x burstable ru_per_sec=4000", true, "CREATE RESOURCE GROUP `x` BURSTABLE RU_PER_SEC = 4000"},
 		{"create resource group x ru_per_sec=20, priority=LOW, burstable", true, "CREATE RESOURCE GROUP `x` RU_PER_SEC = 20 PRIORITY = LOW BURSTABLE"},
+		{"create resource group x ru_per_sec=1000 QUERY_LIMIT=(EXEC_ELAPSED_IN_SEC '10s' ACTION DRYRUN)", true, "CREATE RESOURCE GROUP `x` RU_PER_SEC = 1000 QUERY_LIMIT=(EXEC_ELAPSED_IN_SEC = '10s' ACTION DRYRUN)"},
```

Comment:
The output format is not consistent with each other, `(EXEC_ELAPSED_IN_SEC = '10s' ACTION DRYRUN)` should be `(EXEC_ELAPSED_IN_SEC = '10s' ACTION = DRYRUN)`